src/components/pokemon/Pokemon.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react';
+import axios from "axios";
+import './Pokemon.css';
+
+
+function Pokemon({ endpoint }) {
+  const [ pokemonData, setPokemonData ] = useState({})
```

Comment:
Hier ook, maar dan meer om makkelijker te checken of het nog ingeladen moet worden of niet
```suggestion
  const [ pokemonData, setPokemonData ] = useState();
```